src/Microsoft.IIS.WebManager/angular-cli.json

```diff
@@ -0,0 +1,49 @@
+{

```

Comment:
remove visual studio project files in favor of angular-cli.json since this is a more widely used. My next step would be to remove the self hosting projects since we are not going that route anymore.

Reply:
I believe jimmy is looking for a bit more detailed overview.